### ISSUE TITLE
Highlight code in docs

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/docs-ui#797]
# Description

## User Stories

* > As a user who wants quickly copy & paste examples form the docs, I want them visually highlighted so that I can easily distinguish codeblocks from regular text.

## Value
- usability

## Acceptance Criteria
Style Codeblocks like:
    - background-color: black;
    - color: #4efc54;
    - border-radius:0.7rem;

## Definition of ready
- [ ] Everybody needs to understand the value written in the user story
- [ ] Acceptance criteria have to be defined
- [ ] All dependencies of the user story need to be identified
- [ ] Feature should be seen from an end user perspective
- [ ] Story has to be estimated
- [ ] Story points need to be less than 20

## Definition of done
- Functional requirements
  - [ ] Functionality described in the user story works
  - [ ] Acceptance criteria are fulfilled
- Quality
  - [ ] Code review happened
  - [ ] CI is green (that includes new and existing automated tests)
  - [ ] Critical code received unit tests by the developer
- Non-functional requirements
  - [ ] No sonar cloud issues

## Visual

![Image](https://github.com/owncloud/docs-ui/assets/26610733/2cfe8c49-5208-40d7-9853-23c827c51f02)

**Comment by phil-davis** [comment]
Note: the docs do have a button to copy the code block to the clipboard. That is a really nice feature.
Example that I use is https://doc.owncloud.com/server/next/developer_manual/testing/acceptance-tests.html to copy commands that I need.

But I guess it would be nice that the code blocks really stand out more than they do now.

**Comment by tbsbdr** [comment]
yes, that should be kept. it is just not shown here because I did not hover at the time of the screenshot. no change in that regard for now.
I could imagine to show the button always, not only on hover to be more polite. but this is just a sidenote.
